lib/internal/test_runner/reporter/v8.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const v8 = require('v8');
+const Buffer = require('buffer').Buffer;
+const { serializeError } = require('internal/error_serdes');
+
+
+module.exports = async function* v8Reporter(source) {
+  const serializer = new v8.DefaultSerializer();
+
+  for await (const item of source) {
```

Comment:
is `source` a safe object? meaning, does it have a Symbol.asyncIterator that can't be monkeyed with?

Reply:
this is a readable stream, I am not sure.

Reply:
Makes sense (it is a problem)